qa/rpc-tests/test_framework.py

```diff
@@ -128,10 +128,14 @@ def main(self):
             print("Unexpected exception caught during testing: "+str(e))
             traceback.print_tb(sys.exc_info()[2])
 
-        if not self.options.nocleanup:
-            print("Cleaning up")
+        try:
             stop_nodes(self.nodes)
             wait_bitcoinds()
+        except:
+            pass
```

Comment:
I was thinking: if the client crashed earlier, then stopping the nodes is going to fail, too, and this was the part I wanted to hide. I'm going to take a look at it. :)